modules/virt-running-real-time-checkup.adoc

```diff
@@ -0,0 +1,215 @@
++
+[source,terminal]
+----
+$ oc apply -n <target_namespace> -f <realtime_sa_roles_rolebinding>.yaml
+----
+
+. Create a `ConfigMap` manifest file that contains the input parameters for the checkup:
++
+.Example input config map
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: realtime-checkup-config
+  labels:
+    kiagnose/checkup-type: kubevirt-realtime
+data:
+  spec.timeout: 1h10m # <1>
+  spec.param.vmUnderTestContainerDiskImage: quay.io/kiagnose/kubevirt-realtime-checkup-vm:v0.2.0 # <2>
+  spec.param.oslatDuration: 1h # <3>
+----
+<1> The time, in hours and minutes, before the checkup fails.
+<2> The container disk image for the VM under test.
+<3> The duration for which the OS level thread latency test program runs.
+
+. Apply the `ConfigMap` manifest in the target namespace:
++
+[source,terminal]
+----
+$ oc apply -n <target_namespace> -f <real-time_config_map>.yaml
+----
+
+. Create a `Job` manifest to run the checkup:
++
+.Example job manifest
+[source,yaml,subs="attributes+"]
+----
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: realtime-checkup
+  labels:
+    kiagnose/checkup-type: kubevirt-realtime
+spec:
+  backoffLimit: 0
+  template:
+    spec:
+      serviceAccountName: realtime-checkup-sa
+      restartPolicy: Never
+      containers:
+        - name: realtime-checkup
+          image: registry.redhat.io/container-native-virtualization/kubevirt-realtime-checkup-rhel9:v{product-version}.0
+          imagePullPolicy: Always
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop: [ "ALL" ]
+            runAsNonRoot: true
+            seccompProfile:
+              type: "RuntimeDefault"
+          env:
+            - name: CONFIGMAP_NAMESPACE
+              value: <target-namespace>
+            - name: CONFIGMAP_NAME
+              value: realtime-checkup-config
+            - name: POD_UID
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.uid
+----
+
+. Apply the `Job` manifest in the target namespace:
++
+[source,terminal]
+----
+$ oc apply -n <target_namespace> -f <real-time_job>.yaml
+----
+
+. Wait for the job to complete:
++
+[source,terminal]
+----
+$ oc wait job realtime-checkup -n <target_namespace> --for condition=complete --timeout 1h15m
+----
+
+. Review the results of the checkup by running the following command:
++
+[source,terminal]
+----
+$ oc get configmap realtime-checkup-config -n <target_namespace> -o yaml
+----
++
+.Example output config map (success)
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: realtime-checkup-config
+data:
+# ...
+  status.succeeded: "true" # <1>
+  status.failureReason: "" # <2>
+  status.startTimestamp: "2023-07-31T13:14:38Z" # <3>
+  status.completionTimestamp: "2023-07-31T13:19:41Z" # <4>
+  status.result.vmUnderTestActualNodeName: worker-realtime # <5>
```

Comment:
`worker-realtime` is a label which is applied on the dedicated nodes, rather than a node name.
`<node_name>` maybe? Or something like `worker1`?

Reply:
Changed to `worker1`